extensions/wikia/Recirculation/RecirculationHooks.class.php

```diff
@@ -183,7 +183,8 @@ private static function shoudlNoIndex( $metaDataFromService ) {
 
 		return self::isPrivateOrNotProduction() ||
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1067)